modules/tutorials/pages/java-tutorial/adding-course-enrollments.adoc

```diff
@@ -70,7 +70,7 @@ include::{java-sample-location}AddEnrollments.java[]
 . Retrieve the `art history` course record.
 --
 +
-Each method uses a N1QL call to retrieve a single record from its respective collection.
+Each method uses an {sqlpp} call to retrieve a single record from its respective collection.
```

Comment:
Do we pronounce it "sequel plus plus" or "es queue ell plus plus"? If we make a style rule that it's the former, then we can leave the indefinite article unchanged.

```suggestion
Each method uses a {sqlpp} call to retrieve a single record from its respective collection.
```